_site/index.html

```
@@ -0,0 +1,855 @@
+  
+
+  
+
+</nav>
+
+
+
+<div id="beautifuljekyll-search-overlay">
+
+  <div id="nav-search-exit" title="Exit search">✕</div>
+  <input type="text" id="nav-search-input" placeholder="Search">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)